packages/mobile/src/components/scrubber/usePosition.ts

```diff
@@ -33,7 +33,7 @@ export const usePosition = (
   }, [])
 
   useEffect(() => {
-    let currentTimeout: NodeJS.Timeout
+    let currentTimeout: number
```

Comment:
is there a difference here? im curious

Reply:
I'm not sure what/why it changed but it was flagging TS as an error. I checked the return type of `setTimeout` and it said `number`, I'm thinking maybe the API changed?